public/coffee/ide/editor/directives/aceEditor/auto-complete/AutoCompleteManager.coffee

```diff
@@ -122,6 +125,9 @@ define [
 			range = new Range(end.row, 0, end.row, end.column)
 			lineUpToCursor = @editor.getSession().getTextRange(range)
 			commandFragment = getLastCommandFragment(lineUpToCursor)
+			commandName = getCommandNameFromFragment(commandFragment)
+			if commandName in ['begin', 'end']
```

Comment:
This looks like it would stop the auto-complete running on \begin{| (i.e. creating an environment for the first time. I think we only want to not auto-complete if the \begin{foo} is already complete (i.e. has a closing brace perhaps - although this might not work as a test if we add in auto-closing braces, perhaps a closing brace AND some existing environment name?)